doc/CONTRIBUTING.md

```diff
@@ -5,23 +5,30 @@ Look at open issues, especially with label:
 - [junior job](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22junior+job%22), 
 - [help wanted](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 
+### Makefile devel workflow.
+
+Go in your build dir ```cd $GOPATH/src/github.com/dmacvicar/terraform-provider-libvirt```,
+then  use
+
+```console
+make testacc
+```
+This will build the plugin , performing basic checks (gofmt) and run the acceptance test on your local machine.
+
+
 # Contribute
 
-1. [Build the plugin from source](https://github.com/dmacvicar/terraform-provider-libvirt#building-from-source)
-2. If you have issues, check out the [troubleshooting](https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/doc/TROUBLESHOOTING.md)
-3. Do your code (reference issue on your pr if you fix them) [Closing issues keywords](https://help.github.com/articles/closing-issues-using-keywords/)
-4. Run the acceptance tests.
+1. If you have issues, check out the [troubleshooting](https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/doc/TROUBLESHOOTING.md)
+2. Do your code (reference issue on your pr if you fix them) [Closing issues keywords](https://help.github.com/articles/closing-issues-using-keywords/)
+3. Use the Makefile workflow before submit the PR (building, tests).
```

Comment:
3. Test your code by running the acceptance tests
4. Create a PR

Reply:
done

Reply:
Not done yet.